### ISSUE TITLE
Achievement: Call to Arms!

### ISSUE THREAD

**Issue by Golganeth** [dalaranwow/dalaran-wow#1788]
http://wotlk.openwow.com/achievement=908 (Alliance)
http://wotlk.openwow.com/achievement=909 (Horde)

Since patch 3.3.3, those quests are no longer active, so it is no longer possible to complete them or earn the associated achievement.
However, they should be available at least on Algalon (perhaps enable them on Kirin Tor as well).

List of quests that should be available from the NPCs (Only listed the higher level versions, those which are related to the achievement).

**Alliance:**
- [Call to Arms: Arathi Basin](http://wotlk.openwow.com/quest=11335) - _ID: 11335_
- [Call to Arms: Alterac Valley](http://wotlk.openwow.com/quest=11336) - _ID: 11336_
- [Call to Arms: Eye of the Storm](http://wotlk.openwow.com/quest=11337) - _ID: 11337_
- [Call to Arms: Warsong Gulch](http://wotlk.openwow.com/quest=11338) - _ID: 11338_
- [Call to Arms: Strand of the Ancients](http://wotlk.openwow.com/quest=13405) - _ID: 13405_ ~> Not needed for the achievement
- [Call to Arms: Isle of Conquest](http://wotlk.openwow.com/quest=14163) - _ID: 14163_ ~> Not needed for the achievement

---

**Horde:**
- [Call to Arms: Arathi Basin](http://wotlk.openwow.com/quest=11339) - _ID: 11339_
- [Call to Arms: Alterac Valley](http://wotlk.openwow.com/quest=11340) - _ID: 11340_
- [Call to Arms: Eye of the Storm](http://wotlk.openwow.com/quest=11341) - _ID: 11341_
- [Call to Arms: Warsong Gulch](http://wotlk.openwow.com/quest=11342) - _ID: 11342_
- [Call to Arms: Strand of the Ancients](http://wotlk.openwow.com/quest=13407) - _ID: 13407_ ~> Not needed for the achievement
- [Call to Arms: Isle of Conquest](http://wotlk.openwow.com/quest=14164) - _ID: 14164_ ~> Not needed for the achievement

---

Bear in mind that the available quest should rotate and be completely random, one per day.

**Comment by Aalhazred** [comment]
seems like the npc don't give any quests at all

**Comment by Rushor** [comment]
http://www.wowhead.com/achievement=908/call-to-arms#comments:id=1063845

By eskilajnen (5,712 – 2·13·55) on 2010/06/23 (Patch 3.3.5)
Confirmed by a blue post that this will indeed be obtainable again in the future.

Quoted from the "World of Warcraft Client Patch 3.3.5 Known Issues" list.
"You cannot get the "Call To Arms" Achievement through the Looking for Battleground feature."

http://www.wowhead.com/quest=11341/call-to-arms-eye-of-the-storm#comments:id=1308735
By richert (13,999 – 3·11·99) on 2011/01/25 (Patch 4.0.3)
Although the quest was removed near the end of wrath, it was not removed from quests logs when cata hit, so it will be great for confusing new players.

**Comment by Rushor** [comment]
Game/Battleground: Cast quest complete spell on winning team of call to arms battleground (random questreward, the call to arms quest is different every day)

* note: all quests can now be taken from http://wotlk.openwow.com/npc=15350 or http://wotlk.openwow.com/npc=15351 but the quest will only be completed if it's a BG Weekend and if it's not a random battleground, so you can take all the quests but only the quest that is shown as BG Weekend in the battleground interface will give the quest complete credit if you won the battlegound
* closes: https://github.com/dalaranwow/dalaran-wow/issues/1788
* commit: ad89fe0eb945214bb1e7661508ced431df0f508d

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
